### ISSUE TITLE
Can't load NES ROM. Fades to black and then back to menu.

### ISSUE THREAD

**Issue by TheVrolok** [Aloshi/EmulationStation#479]
Only output from emulationstation is (without the periods):

==============================================.
==============================================.

Log file is:

lvl2:   EmulationStation - v2.0.1a, built Jul 21 2015 - 19:40:05
lvl2:   Creating surface...
lvl2:   Created window successfully.
lvl2:   Added known joystick USB,2-axis 8-button gamepad   (instance ID: 0, device index: 0)
lvl2:   Checking available OpenGL extensions...
lvl2:    ARB_texture_non_power_of_two: MISSING
lvl2:   Loading system config file /home/pi/.emulationstation/es_systems.cfg...
lvl2:   Parsing XML file "/home/pi/.emulationstation/gamelists/nes/gamelist.xml"...
lvl1:   File ""/home/pi/games/ROMs/NES/1942 (U) [!].nes"" does not exist! Ignoring.
lvl2:    req sound [basic.launch]
lvl2:      (missing)
lvl2:   Attempting to launch game...
lvl2:           ~/games/RetroArch/retroarch -L ~/games/RetroArch/cores/libretro-fceumm.so /home/pi/games/ROMs/NES/1942.nes
lvl2:   Creating surface...
lvl2:   Created window successfully.
lvl2:   Added known joystick USB,2-axis 8-button gamepad   (instance ID: 0, device index: 0)
lvl2:   Added known joystick USB,2-axis 8-button gamepad   (instance ID: 0, device index: 0)

I don't even know where to start with no actual error messages.

This is on my RPi2 with Raspbian (not RetroPi), compiled ES from the RPI-standalone tutorial on the site. RetroArch runs the game fine outside of ES, if I just run "~/games/RetroArch/retroarch -L ~/games/RetroArch/cores/libretro-fceumm.so /home/pi/games/ROMs/NES/1942.nes" it starts up no problems.

**Comment by joolswills** [comment]
Doesn't look like an ES issue to me. Try with full paths without the tilde ~

**Comment by TheVrolok** [comment]
No change. Appreciate the assistance, though.

**Comment by joolswills** [comment]
post a log again please - (via pastebin or similar)

**Comment by joolswills** [comment]
and your emulationstation config please.

**Comment by TheVrolok** [comment]
<systemList>
        
        <system>

                
                <name>nes</name>

                
                <fullname>Nintendo Entertainment System</fullname>

                
                <path>~/games/ROMs/NES</path>

                
                <extension>.nes .NES</extension>

                
                <command>/home/pi/games/RetroArch/retroarch -L /home/pi/games/RetroArch/cores/libretro-fceumm.so %ROM%</command>


                
                <platform>nes</platform>

                
                <theme>nes</theme>
        </system>
</systemList>

**Comment by joolswills** [comment]
and

```
/home/pi/games/RetroArch/retroarch -L /home/pi/games/RetroArch/cores/libretro-fceumm.so /home/pi/games/ROMs/NES/SMB3.nes
```

def works from console?

If you can't get this sorted btw you could just use retropie (you can customise/trim it down as needed).

**Comment by TheVrolok** [comment]
Yep, copy/pasted it to make sure. Maybe something from the manual compile? I don't have much else on the RPi, wouldn't be hard to format/stick RetroPi on to get it working and then add the other things I'm using (LAMP stack, etc.).

**Comment by sub-atomic** [comment]
This is happening to me too on Windows 8.1. My config:

```
<system>
	    <name>nes</name>
	    <fullname>Nintendo Entertainment System</fullname>
 	   <path>C:\Games\RETRO\Nintendo NES</path>
 	   <extension>.nes .NES .fds</extension>
 	   <command>C:\Games\RETRO\Emulators\RetroArch\retroarch.exe -L C:\Games\RETRO\Emulators\RetroArch\cores\nestopia_libretro.dll "%ROM_RAW%"</command>
 	   <platform>nes</platform>
 	   <theme>nes</theme>
	</system>
```
